server.js

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import mongoose from 'mongoose';
+const server = express();
+import ApiRouter from './api/registration.js';
+import passportRoutes from './passportConfig';
+
+const bodyParser = require('body-parser');
+server.use(express.static('public'));
+server.use(bodyParser.urlencoded({ extended: false }));
+server.use(bodyParser.json());
+server.set('views', './views');
+server.set('view engine', 'ejs');
+server.use('/api', ApiRouter);
+server.use('/', passportRoutes);
+
+server.get('/', allowIfLoggedOut, (req, res) => {
+    res.redirect('/home/about');
+});
+
+server.get('/home*', allowIfLoggedOut, (req, res) => {
+    res.render('home');
```

Comment:
what is this? with react, you shouldn't need to render a view for react to work.